devops/providers/azure-devops/templates/infrastructure/README.md

```diff
@@ -176,30 +176,45 @@ There should be one Azure Unified pipeline for each individual application. This
+
+### Global Variable Group
+
+This group should be named `Infrastructure Pipeline Variables`. It will need the following variables defined:
+
+| Variable Name | Description | Sample Value |
+| ------------- | ------------- | ------------- |
+| `AGENT_POOL` | The Azure DevOPS agent pool name | `Hosted Ubuntu 1604` |
+| `BUILD_ARTIFACT_NAME` | The build artifact naming prefix | `drop` |
+| `BUILD_ARTIFACT_PATH_ALIAS` | Alias for build artifact | `artifact` |
+| `GO_VERSION` | The Go version to use for the automated test steps | `1.12.5` |
+| `PIPELINE_ROOT_DIR` | The relative path of the Azure DevOPS CI/CD pipeline directory | `devops/providers/azure-devops/templates/infrastructure` |
+| `REMOTE_STATE_CONTAINER` | The remote state Azure storage container name | `cobaltfstate-remote-state-container` |
+| `SCRIPTS_DIR` | The directory name containing the scripts used for the Azure DevOPS pipeline | `scripts` |
+| `ARM_PROVIDER_STRICT` | Whether or not to enable strict mode in terraform [`true`|`false`] | `true` |
+| `TEST_HARNESS_DIR` | Relative directory of test harness | `test-harness/`
+| `TF_DEPLOYMENT_TEMPLATE_ROOT` | Relative directory of template to deploy | `infra/templates/ |az-isolated-service-single-region` |
+| `TF_ROOT_DIR` | Relative directory of terraform root | `infra/` |
+| `TF_VERSION` | Version of terraform | `0.12.4` |
+| `TF_WARN_OUTPUT_ERRORS` | Flag for terraform that results in no failures for a warning | 1 |
+
+### Stage Specific Pipeline Variables
+
+This group should be named `$STAGE Environment Variables`. It will need the following variables defined:
+
+
+| Variable Name | Description | Sample Value |
+| ------------- | ------------- | ------------- |
+| `ARM_SUBSCRIPTION_ID` | The Azure subscription of the service principal used for the deployment | 49e3a7a8-c63e-4124-948c-ee51b74d5801 |
+| `REMOTE_STATE_ACCOUNT` | The Azure storage account for remote terraform state | cobaltfstate |
+| `SERVICE_CONNECTION_NAME` | The azure devops service connection name to use for the Terraform deployments | Cobalt Deployment Administrator |
 
 ## Security
 
 Deployments are carried out by service principals. The build pipeline references SP credentials from a keyvault-backed variable group.
 
 You'll need to create a keyvault resource that includes the above variables where `Variable Source` is set to `Key-Vault`.
 
-Follow these [instructions](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml#link-secrets-from-an-azure-key-vault) to associate all the above secrets to a variable group called `KV Secrets`.
\ No newline at end of file
+Follow these [instructions](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml#link-secrets-from-an-azure-key-vault) to associate all the above secrets to a variable group called `KV Secrets`.
```

Comment:
This is a good document that you link to. Seems complete with setting up access policy, etc...

Reply:
This is actually no longer needed since all secrets are sourced from the service connection. I'll remove the step